sql/moz-fx-data-shared-prod/org_mozilla_fenix_nightly/baseline_clients_daily/view.sql

```diff
@@ -1,8 +0,0 @@
--- Generated via bigquery_etl.glean_usage
```

Comment:
The existing checked-in content for `org_mozilla_fenix_nightly` is stale. In practice, it's ignored by the baseline ETL code in this repository. We remove it now to reduce confusion.